www/artists/arocamora/figures.html

```diff
@@ -4,62 +4,82 @@
+                        </td>
+                    </tr>
+                    <tr>
+                        <th>
+                            25 YEARS OF WATCHING HER GROW: handbuilt porcelain, oxides, welded steel
+                        </th>
+                        <th>
+                            Witnesses: Sculpture, stoneware, oxides, under glazes, acrylics, 2006
+                        </th>
+                        <th>
+                          FIGURE WITH TWO RABBITS: white stoneware, underglazes, terra sigilatta
+
+                        </th>
+
+                    </tr>
+
+                </table>
                 <br>
-                25 YEARS OF WATCHING HER GROW: handbuilt porcelain, oxides, welded steel
-            </p>
-            <h2>
                 Figures are pictorial or sculptural representations of many things, especially of the human body.
-            </h2>
-            <p>
-                <a href="images/figure02.jpg"><img src="images/figure02.jpg" alt="figure"></a>
                 <br>
-                Witnesses: Sculpture, stoneware, oxides, under glazes, acrylics, 2006
-            </p>
-            <p>
-                <a href="images/figure01.jpg"><img src="images/figure01.jpg" alt="vessel"></a>
                 <br>
-                FIGURE WITH TWO RABBITS: white stoneware, underglazes, terra sigilatta
-            </p>
-        </article>
-        <section>
-          <ul>
-              <li>
-                  <a href="../">Sask Artists</a>
-              </li>
-              <li>
-                  <a href="vessels.html">Vessels</a>
-              </li>
-              <li>
-                  <a href="figures.html">Figures</a>
-              </li>
-              <li>
-                  <a href="index.html">Home</a>
-              </li>
-              <li>
-                  <a href="sculpture.html">Sculpture</a>
-              </li>
-          </ul>
-        </section>
-        <footer>
-            <p>
-                Created by Anna Tunison 2010
-            </p>
-        </footer>
+                Anita Rocamora is visual artist who works in ceramics and mixed media. Anita uses natuer and humanity for insperation in her art work. Anita uses deferent shapes from natuer and put them in a shape for a vase. Anita trys to create objects that are entirely imaginary yet evoke a feeling of familiarity and recognition. In nature. Form and function flow from one another in even the most absurdly extravagant creations. The focus of her work is to exploration other shapes that no one ever used. Anitas primary materials are porcelian and stoneware. Clay contains traces of all that has existed on this planet. From single-cell organisms to mountains. To ground together by erosion and geological forces to create a material that can be transformed at will. Anita likes to make with these material objects for facilitate joy, awareness, whimsy, and a sense of connection to the earth.
```

Comment:
Why did you copy this text from the index? It seems unnecessary, and it includes a number of spelling errors that are a bit obvious (natuer?)